Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -748,12 +749,13 @@ Sends a new chat message in the specified chat.
 `microsoft-teams-message-send-to-chat`
 
 ##### Required Permissions
-`ChatMessage.Send` - Delegated
-`Chat.ReadWrite` - Delegated
-`TeamsAppInstallation.ReadWriteForChat` - Delegated
-`TeamsAppInstallation.ReadWriteSelfForChat` - Delegated
-`TeamsAppInstallation.ReadWriteSelfForChat.All` - Application               
-`TeamsAppInstallation.ReadWriteForChat.All` - Application
+`ChatMessage.Send` - Delegated<br>
+`Chat.ReadWrite` - Delegated<br>
+`TeamsAppInstallation.ReadWriteForChat` - Delegated<br>
+`TeamsAppInstallation.ReadWriteSelfForChat` - Delegated<br>
+`TeamsAppInstallation.ReadWriteSelfForChat.All` - Application<br>               
+`TeamsAppInstallation.ReadWriteForChat.All` - Application<br>
+`AppCatalog.Read.All` - Application
```

Comment:
```suggestion
`ChatMessage.Send` - Delegated
`Chat.ReadWrite` - Delegated
`TeamsAppInstallation.ReadWriteForChat` - Delegated
`TeamsAppInstallation.ReadWriteSelfForChat` - Delegated
`TeamsAppInstallation.ReadWriteSelfForChat.All` - Application
`TeamsAppInstallation.ReadWriteForChat.All` - Application
`AppCatalog.Read.All` - Application
```

Reply:
As discussed on Slack this is on purpose to add line breaks.